argh.h

```diff
@@ -382,8 +388,11 @@ namespace argh
          auto optIt = params_.find(trim_leading_dashes(name));
          if (params_.end() != optIt)
             return string_stream(optIt->second);
-      }      
+      }
       std::ostringstream ostr;
+      using value_type = typename std::decay<T>::type;

```

Comment:
Please add a comment explaining what you are doing in the next 3 lines and why.

Reply:
Should I add a comment in the code? References should be removed from the type `T` in order to use type trait `is_floating_point`. Unfortunately, numeric limits (sometimes) cannot be correctly determined for arrays, thus one should decay the type before.

But now I am not sure that it is actually necessary, the simplest way is to set the maximal precision like `ostr.precision(std::numeric_limits<long double>::max_digits10);` It is only one line and it solves all the problems. What do you think?